blockstore/badger/blockstore.go

```diff
@@ -131,6 +131,15 @@ func (b *Blockstore) Close() error {
 	return b.DB.Close()
 }
 
+// GC runs garbage collection on the value log
+func (b *Blockstore) GC() error {
```

Comment:
I'd call this CollectGarbage to match the go-datastore method

Reply:
ok, will do.

Reply:
done.